app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1090,104 +1178,172 @@ def validate_feature_value(self, value):
+        source="targeting_config_slug",
+        choices=NimbusExperiment.TargetingConfig.choices,
+        required=True,
+    )
+    referenceBranch = NimbusBranchReviewSerializer(
+        source="reference_branch", required=True
+    )
+    treatmentBranches = NimbusBranchReviewSerializer(
+        source="treatment_branches", many=True
     )
-    reference_branch = NimbusBranchReviewSerializer(required=True)
-    treatment_branches = NimbusBranchReviewSerializer(many=True)
-    feature_config = serializers.PrimaryKeyRelatedField(
+    featureConfig = serializers.PrimaryKeyRelatedField(
         queryset=NimbusFeatureConfig.objects.all(),
         allow_null=False,
         error_messages={"null": NimbusConstants.ERROR_REQUIRED_FEATURE_CONFIG},
         write_only=True,
     )
-    feature_configs = serializers.PrimaryKeyRelatedField(
+    featureConfigs = serializers.PrimaryKeyRelatedField(
+        source="feature_configs",
         queryset=NimbusFeatureConfig.objects.all(),
         many=True,
         allow_empty=False,
         error_messages={"empty": NimbusConstants.ERROR_REQUIRED_FEATURE_CONFIG},
     )
-    primary_outcomes = serializers.ListField(
-        child=serializers.CharField(), required=False
+    primaryOutcomes = serializers.ListField(
+        source="primary_outcomes", child=serializers.CharField(), required=False
     )
-    secondary_outcomes = serializers.ListField(
-        child=serializers.CharField(), required=False
+    secondaryOutcomes = serializers.ListField(
+        source="secondary_outcomes", child=serializers.CharField(), required=False
     )
-    risk_partner_related = serializers.BooleanField(
+    riskPartnerRelated = serializers.BooleanField(
+        source="risk_partner_related",
         required=True,
         allow_null=False,
         error_messages={"null": NimbusConstants.ERROR_REQUIRED_QUESTION},
     )
-    risk_revenue = serializers.BooleanField(
+    riskRevenue = serializers.BooleanField(
+        source="risk_revenue",
         required=True,
         allow_null=False,
         error_messages={"null": NimbusConstants.ERROR_REQUIRED_QUESTION},
     )
-    risk_brand = serializers.BooleanField(
+    riskBrand = serializers.BooleanField(
+        source="risk_brand",
         required=True,
         allow_null=False,
         error_messages={"null": NimbusConstants.ERROR_REQUIRED_QUESTION},
     )
+    isFirstRun = serializers.BooleanField(source="is_first_run")
+    isRollout = serializers.BooleanField(source="is_rollout")
+    isSticky = serializers.BooleanField(source="is_sticky")
+    riskMitigationLink = serializers.URLField(
+        source="risk_mitigation_link",
+        min_length=0,
+        max_length=255,
+        required=False,
+        allow_blank=True,
+    )
+    warnFeatureSchema = serializers.BooleanField(source="warn_feature_schema")
 
     class Meta:
         model = NimbusExperiment
-        exclude = ("id",)
+        fields = [
+            "application",
```

Comment:
I think we should write this too in a sorted way 🤔

Reply:
and I think everywhere where we have `fields`

Reply:
Agree, let's do it as a followup https://github.com/mozilla/experimenter/issues/7735